src/self-managed-debug/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
```

Comment:
I manually created this file by copying `.toml`s from other projects. Wondering if there's an easier way to do this though via CLI commands.

Reply:
That's what I do. You start with some similar crate and use it as a starting point.